### ISSUE TITLE
$RENEWED_LINEAGE equivalent on acme.sh

### ISSUE THREAD

**Issue by alexandre1985** [Neilpang/acme.sh#1534]
I am running a script on `--renew-hook` and that script is this:
```
#!/bin/bash
# Full path to pre-generated Diffie Hellman Parameters file
dhparams=/etc/hitch/dhparams.pem

if [[ "${RENEWED_LINEAGE}" == "" ]]; then
    echo "Error: missing RENEWED_LINEAGE env variable." >&2
    exit 1
fi

umask 077
cat ${RENEWED_LINEAGE}/test.com.key \
${RENEWED_LINEAGE}/fullchain.cer \
${dhparams} > ${RENEWED_LINEAGE}/hitch-bundle.pem
```
The script isn't doing what it is supposed to do (can't find hitch-bundle.pem). I believe the problem seems that `${RENEWED_LINEAGE}` is a certbot variable only, and in `acme.sh` I need a similar variable to get the directory of the certificates

**Comment by farces** [comment]
That's right - during an _--issue_ action the _--renew-hook_ script isn't run as technically the certificate is only issued, not renewed. 

It's not ideal but what I do when issuing a new certificate is immediately force a renewal with _--renew --force_ to trigger the renew hook. You can also run your  script manually the first time from the certificate directory.

FYI if `$(pwd)` is correct you can replace your `${acmehome}/${domain}/fullchain.cer` code with just `$(pwd)/fullchain.cer` and not have to hard-code your acmehome directory in the script, if portability is important e.g. if you decide to run under Docker.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
